.github/workflows/nessie.yaml

```diff
@@ -287,6 +306,32 @@ jobs:
         run: |
           docker-compose -f nessie/ops/docker-compose.yaml down -v
           docker-compose -f nessie/ops/docker-compose.yaml up --quiet-pull --exit-code-from=nessie
+
+  run-system-gcp-gs:
+    name: Run latest lakeFS app on Google Cloud Platform and Google Cloud Storage
+    needs: [gen-code, deploy-image]
+    runs-on: ubuntu-20.04
+    env:
+      TAG: ${{ needs.deploy-image.outputs.tag }}
+      # Setting Account_ID as a secret as a way to avoid specifying it here
+      REPO: ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.us-east-1.amazonaws.com
+    steps:
+      - name: Retrieve generated code
+        uses: actions/download-artifact@v2
+        with:
+          name: generated-code
+          path: /tmp/
+      - name: Unpack generated code
+        run: tar -xf /tmp/generated.tar.gz
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
```

Comment:
Why do we need aws creds for gs?

Reply:
Fetch from ECR.  AFAIK that repo is not open for public read.  E.g. if I don't do the `aws ecr get-login ...; docker login` dance then Docker cannot fetch images:
```sh
ariels@redqueen:~/dev/lakeFS$ docker pull 977611293394.dkr.ecr.us-east-1.amazonaws.com/lakefs:latest
Error response from daemon: Head https://977611293394.dkr.ecr.us-east-1.amazonaws.com/v2/lakefs/manifests/latest: no basic auth credentials
```

We might ditch ECR (for this, at least) and just use the GitHub Docker registry.  But that would be in a separate PR.